.github/workflows/flow.yml

```diff
@@ -41,7 +41,32 @@ jobs:
     - uses: actions/checkout@v4
     - uses: psf/black@stable
 
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+        fetch-depth: 0
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: 3.8
+
+    - name: Install build
```

Comment:
Might be worth replacing all this with https://github.com/hynek/build-and-inspect-python-package?